background/nudge/activities.go

```diff
@@ -154,3 +134,28 @@ func (n *NudgeWorker) NotifySymptomFollowUpActivity(ctx context.Context, account
 
 	return n.mongo.UpdateAccountSymptomNudge(accountNumber)
 }
+
+// NotifySymptomSpikeActivity send notifications to accounts those have symptoms spiked around
+func (n *NudgeWorker) NotifySymptomSpikeActivity(ctx context.Context, accountNumber string, symptoms []schema.Symptom) error {
+	logger := activity.GetLogger(ctx)
+
+	logger.Info("Prepare the message context for following up symptoms", zap.Any("symptoms", symptoms))
+
+	var symptomsIDs = make([]schema.SymptomType, 0)
+	for _, s := range symptoms {
+		symptomsIDs = append(symptomsIDs, s.ID)
+	}
+
+	headings, contents, err := SymptomListingMessage("symptom_spike", symptoms)
+	if err != nil {
+		logger.Error("can not generate symptoms follow-up message", zap.Error(err))
+	}
+
+	return n.Background.NotifyAccountByText(accountNumber,
```

Comment:
What will happen if err != nil ? 
It seems no nil/empty checking in the NotifyAccountByText

Reply:
It returns the error so it will be checked outside.